turbo.json

```diff
@@ -1,135 +1,16 @@
 {
```

Comment:
I guess we need to keep the following scripts since we have them in all packages:
- build:clean
- lint
- typecheck